images/TEMPLATE/config/template.apko.yaml

```diff
@@ -0,0 +1,19 @@
+contents:
```

Comment:
Can we add comments here on how to test if you have locally built packages? In the document, I had the example, of adding the keyrings, local package dir, etc.

At least the workflow I've been using for things that include many packages that need to be built, I keep on iterating on packages, and then test them locally.

Or maybe add that in the README.md file, or does it just magically work like this? 🤔

Reply:
I think that's somewhat orthogonal to this change, but I think we should document it in `TERRAFORM.md` in the root of the repo.